step-templates/terraform-custom-script.json

```diff
@@ -0,0 +1,109 @@
+      "Octopus.Action.SubstituteInFiles.TargetFiles": "InfrastructurePackage\\*.tf\nInfrastructurePackage\\*.tfvars"
+    },
+    "Parameters": [
+      {
+        "Id": "0170e094-b282-4924-b1ed-5103015e366f",
+        "Name": "TerraformPackageId",
```

Comment:
I can't see this Parameter referenced in the script? Was it by chance replaced with the `InfrastructurePackage` parameter? If so, I think the `TerraformPackageId` parameter can be safely removed.

Reply:
The package variable is required because the step has a "Referenced Package" so that the project can select a package and then it is referenced in the script as InfrastructurePackage.

![image](https://user-images.githubusercontent.com/6273429/111202302-9d567680-85bb-11eb-8434-c99601983d80.png)

Reply:
Thanks @alastairtree - not sure how I missed it referenced: `"PackageParameterName": "TerraformPackageId"` :smile: